tests/subsys/bootloader/bl_crypto/src/main.c

```diff
@@ -159,6 +158,8 @@ void test_sha256(void)
 
 	/* Size restrictions. */
 #if CONFIG_FLASH_SIZE > 300
+	test_sha256_string(const_fw_data, ARRAY_SIZE(const_fw_data),
+				image_fw_hash, true);
```

Comment:
I believe that `TAB`s are used for first indentation, and then `spaces` for final alignment of `image_fw_hash` to align with `const_fw_data`.